core/indigo-core/molecule/src/molecule_cdxml_saver.cpp

```diff
@@ -1367,6 +1367,12 @@ void MoleculeCdxmlSaver::addMetaObject(const MetaObject& obj, int id)
                 s->SetAttribute("font", 4);
                 s->SetAttribute("size", font_size / kCDXMLFonsSizeMultiplier);
                 s->SetAttribute("face", font_face.face);
+                if (font_face.is_superscript)
+                {
+                    s->SetAttribute("face", "64");
```

Comment:
move  "64" to constant

Reply:
Added two constans KCDXMLFontStyleSuperscript and KCDXMLFontStyleSubscript. The constants in the CDXML format are described as font style with names subscript and superscript of UINT16 type, so I have taken that into concideration declaring these variables in Indigo